packages/webapp/src/locales/en/rrule.json

```diff
@@ -0,0 +1,49 @@
+{
+  "__comment": "THIS IS A FILE COMMENT FOR rrule.json. This locale file is not for use in 'i18n' and only for use with 'rrule' software package. The english file is only used as a base for translation to other languages -- DO NOT MODIFY.",
```

Comment:
Sorry why do we need this? CrowdIn translators can't modify the source file anyway and there is a high likelihood of someone translating these instructions if we put them up. As for devs, do you think there is more danger of the files getting modified in their new location than there was in their previous location?